### PR TITLE
fix(cloud-service-log): improve log tab 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
[Cloud Service Detail > Log tab]
- fix style bug
- get all data at first loading (& do not refresh when page number changed)
- get different time list by page

### CODE REVIEW

File: src/services/asset-inventory/cloud-service/cloud-service-detail/modules/CloudServiceLogTab.vue [hunk 1]
```diff
@@ -135,27 +138,50 @@ export default defineComponent<Props>({
...
+                        name: 'last3months',
+                        label: i18n.t('INVENTORY.CLOUD_SERVICE.HISTORY.DETAIL.LOG_TAB.LAST_3_MONTHS'),
+                        start: _date.subtract(3, 'month'),
+                        end: _date,
+                    },
+                ];
+                if (props.date) return timeWithinList.filter((d) => HISTORY_LOG_PERIOD_LIST.includes(d.name));
+                return timeWithinList.filter((d) => CLOUD_SERVICE_LOG_PERIOD_LIST.includes(d.name));
+            }),
+            selectedTimeWithin: props.date ? 'last1day' : 'last1week',
```
Comment (WANZARGEN) [inline src/services/asset-inventory/cloud-service/cloud-service-detail/modules/CloudServiceLogTab.vue:184]: suggestion: What about to use constants for consistency?

```suggestion
            selectedTimeWithin: props.date ? HISTORY_LOG_PERIOD_LIST[0] : CLOUD_SERVICE_LOG_PERIOD_LIST[0],
```

File: src/services/asset-inventory/cloud-service/cloud-service-detail/modules/CloudServiceLogTab.vue [hunk 2]
```diff
@@ -215,14 +241,13 @@ export default defineComponent<Props>({
         // handler
         const dynamicLayoutListeners: Partial<DynamicLayoutEventListener> = {
             fetch(options) {
-                if (options?.pageStart) {
-                    state.pageStart = options.pageStart;
+                if (options?.pageStart) state.pageStart = options.pageStart;
+                if (options?.pageLimit) state.pageLimit = options.pageLimit;
+                if (!state.searchText && !options?.searchText) return;
```
Comment (WANZARGEN) [inline src/services/asset-inventory/cloud-service/cloud-service-detail/modules/CloudServiceLogTab.vue:246]: issue: I think this statement cause the problem that blocking refresh action.

**Review by WANZARGEN — APPROVED** [review]
I left some comments so please check it!
Thanx for your hard work!

**Review by sulmoJ — APPROVED** [review]
Has the service plan changed again?
Thank you for your hard work! (Thank you also wanjin)

**Pull request merged**